lightstep/constants.py

```diff
@@ -24,3 +24,7 @@
 
 # utils constants
 SECONDS_TO_MICRO = 1000000
+
+# Recorder constants
+MAX_LOG_MESSAGE_LEN = 1024
+LONGEST_POSSIBLE_SUB_LOG = 984
```

Comment:
Sorry, I think the clumsy naming is the source of the confusion (fixed). MAX_LOG_MESSAGE_LEN is the most memory I'm allowing to be allocated to a log message, so if a message is >1024 bits then I truncate it to its first 984 characters (size of python string size in bits = 40 + #ofCharacters). It helps avoid using magic numbers.